armi/operators/operator.py

```diff
@@ -618,6 +633,58 @@ def interactAllCoupled(self, coupledIteration):
         activeInterfaces = [ii for ii in self.interfaces if ii.enabled()]
         self._interactAll("Coupled", activeInterfaces, coupledIteration)
 
+    def _tightCouplingOldValues(self):
+        activeInterfaces = [ii for ii in self.interfaces if ii.enabled()]
+        for interface in activeInterfaces:
+            interface.tightCouplingOldValue = interface.getTightCouplingValue()
+
+    def _computeTightCouplingConvergence(self):
+        activeInterfaces = [ii for ii in self.interfaces if ii.enabled()]
```

Comment:
This could be:

```python
if ii.enabled() and ii.tightCouplingOldValue is not None
```

It would save you an indent in that next loop.

Reply:
This was addressed somewhat in https://github.com/terrapower/armi/pull/1033/commits/668f87a709593a67b54f10ec15b4087936442f64. Let me know what you think?

Reply:
That commit is only touches a unit test and not this file.

Reply:
Okay, so am I crazy or... has the method I mentioned above (`_computeTightCouplingConvergence()`) gone away, but we still have a test `test_computeTightCouplingConvergence()`?

Reply:
This commit e01ecd0f21ec6d2911cf6485e633dd09646f8c5e looks be more in line with this thread. Sorry about that.